packages/flutter/lib/src/material/text_selection.dart

```diff
@@ -136,6 +136,7 @@ class _MaterialTextSelectionControls extends TextSelectionControls {
   Widget buildToolbar(
     BuildContext context,
     Rect globalEditableRegion,
+    double textlineHeight,
```

Comment:
This line breaks the current API.

Reply:
Retracting my idea about using `textLineHeight` as an instance variable after talking it over.